src/components/volume/volume.js

```diff
@@ -23,38 +23,40 @@ const mapStateToProps = state => ({
   isMobile: state.shell.isMobile
 });
 
+const COMPONENT_NAME = 'volume';
```

Comment:
also, all component names start with capital, so please align, same for vrStereo component

Reply:
done